easybuild/easyconfigs/m/matplotlib/matplotlib-1.4.3-intel-2015b-Python-3.5.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'matplotlib'
+version = '1.4.3'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = [('matplotlib-%(version)s_Qhull-intel-fix.patch')]
+
+python = 'Python'
+pythonversion = '3.5.0'
+pyshortver = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('freetype', '2.5.5'),
+    ('libpng', '1.6.16'),
```

Comment:
same matplotlib version as existing ones, I won't bump these versions until there's a newer version of matplotlib